Improper handling of session API

In setting up a py.test session fixture, I ran into an issue where Alchy does not properly handle arguments passed from SQLAlchemy.

```
>       session.add(model)

tests/test_models.py:17:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../../.envs/platform/lib/python3.5/site-packages/sqlalchemy/orm/scoping.py:157: in do
    return getattr(self.registry(), name)(*args, **kwargs)
../../../.envs/platform/lib/python3.5/site-packages/sqlalchemy/util/_collections.py:1025: in __call__
    val = self.registry.value = self.createfunc()
../../../.envs/platform/lib/python3.5/site-packages/alchy/manager.py:241: in create_session
    return self.session_class(self, **options)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <alchy.session.Session object at 0x10ddaf908>, manager = <alchy.manager.Manager object at 0x10c512f28>, options = {'binds': {}}, bind = <sqlalchemy.engine.base.Connection object at 0x10dda2a20>

    def __init__(self, manager, **options):
        self.manager = manager
        bind = options.pop('bind', manager.engine)
        super(Session, self).__init__(bind=bind,
                                      binds=manager.binds_map,
>                                     **options)
E       TypeError: __init__() got multiple values for keyword argument 'binds'

../../../.envs/platform/lib/python3.5/site-packages/alchy/session.py:22: TypeError
```

The easy fix here is to update `__init__` within the Session class to pop off the extraneous options.  That said, I am not sure what the ramifications are of ignoring the `binds` value.  Therefore, my proposed solution is as follows:

```
    def __init__(self, manager, **options):
        self.manager = manager
        bind = options.pop('bind', manager.engine)
        binds = options.pop('binds', {}) or manager.binds_map
        super(Session, self).__init__(bind=bind,
                                      binds=binds,
                                      **options)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,22 +1,21 @@
-# Config file for automatic testing at travis-ci.org
 sudo: false
-
 language: python
-
 python:
   - "2.7"
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "nightly"  # points to 3.7 right now (2016-12-28)
 
-# command to install dependencies
+matrix:
+  fast_finish: true
+  allow_failures:
+    - python: "3.6"
+    - python: "nightly"
+
 install:
   - make travisci-install
-
-# command to run tests
 script:
   - make travisci-test
-
-# tasks to execute after successful test
 after_success:
-  - pip install python-coveralls
   - coveralls

--- a/AUTHORS.rst
+++ b/AUTHORS.rst
@@ -13,3 +13,4 @@ Contributors
 - Jeff Knupp, http://www.jeffknupp.com/,  `jeffknupp@github <https://github.com/jeffknupp>`_
 - Seth P., `seth-p@github <https://github.com/seth-p>`_
 - Eli Chan, eli.d.chan@gmail.com, `elidchan@github <https://github.com/elidchan>`_
+- Brian Bruggeman, brian.m.bruggeman@gmail.com, `brianbruggeman@github <https://github.com/brianbruggeman>`_

--- a/alchy/session.py
+++ b/alchy/session.py
@@ -12,14 +12,17 @@ __all__ = [
 class Session(SessionBase):
     """The default session used by :class:`alchy.manager.Manager`. It extends
     the default session system with bind selection.
+
+    Args:
+        manager(alchy.manager.Manager): Alchy manager instance
+        options(dict): pass-through to SessionBase call
     """
 
     def __init__(self, manager, **options):
         self.manager = manager
         bind = options.pop('bind', manager.engine)
-        super(Session, self).__init__(bind=bind,
-                                      binds=manager.binds_map,
-                                      **options)
+        binds = options.pop('binds', {}) or manager.binds_map
+        super(Session, self).__init__(bind=bind, binds=binds, **options)
 
     def get_bind(self, mapper=None, clause=None):
         """Return engine bind using mapper info's bind_key if present."""

